api/query/atoms_test.go

```diff
@@ -919,11 +974,11 @@ func TestStructuredQuery_bindExistsWithTwoProducts(t *testing.T) {
 			})
 		if i%2 == 0 { // Evens are edge
 			or1.Args = append(or1.Args,
-						RunTestStatusEq{
-							Run:    int64(i),
-							Status: 1,
-						},
-				)
+				RunTestStatusEq{
```

Comment:
Note for reviewers: this was done automatically as part of go fmt. Nothing was changed here